tsl/test/isolation/specs/continuous_aggs_multi.spec

```diff
@@ -0,0 +1,54 @@
+
+setup
+{
+    SELECT _timescaledb_internal.stop_background_workers();
+    CREATE TABLE ts_continuous_test(time INTEGER, val INTEGER);
+    SELECT create_hypertable('ts_continuous_test', 'time', chunk_time_interval => 10);
+    INSERT INTO ts_continuous_test SELECT i, i FROM
+        (SELECT generate_series(0, 29) AS i) AS i;
+    CREATE VIEW continuous_view_1( bkt, cnt)
+        WITH ( timescaledb.continuous, timescaledb.refresh_lag = '-5', timescaledb.refresh_interval='72 hours')
+        AS SELECT time_bucket('5', time), COUNT(val)
+            FROM ts_continuous_test
+            GROUP BY 1;
+    CREATE VIEW continuous_view_2(bkt, maxl)
+        WITH ( timescaledb.continuous,timescaledb.refresh_lag='-10',  timescaledb.refresh_interval='72 hours')
+        AS SELECT time_bucket('5', time), max(val)
+            FROM ts_continuous_test
+            GROUP BY 1;
+}
+
+teardown {
+    DROP TABLE ts_continuous_test CASCADE;
+}
+
+session "I"
+step "I1"	{ INSERT INTO ts_continuous_test SELECT 0, i*10 FROM (SELECT generate_series(0, 10) AS i) AS i; }
+
+step "S1"	{ SELECT count(*) FROM ts_continuous_test; }
+
+session "R1"
+step "Refresh1"	{ REFRESH MATERIALIZED VIEW continuous_view_1; }
+
+session "R1_sel"
+step "Refresh1_sel"	{ select * from continuous_view_1 where bkt = 0; }
+
+session "R2"
+step "Refresh2"	{ REFRESH MATERIALIZED VIEW continuous_view_2; }
+
+session "R2_sel"
+step "Refresh2_sel"	{ select * from continuous_view_2 where bkt = 0; }
+
+#the completed threshold will block the REFRESH from writing 
+session "LC"
+step "LockCompleted" { BEGIN; LOCK TABLE _timescaledb_catalog.continuous_aggs_completed_threshold IN SHARE MODE; }
+step "UnlockCompleted" { ROLLBACK; }
+
+
+#refresh1 and refresh2 see the same invalidated data
+permutation "Refresh1" "Refresh2" "I1" "Refresh1" "Refresh2" "Refresh1_sel" "Refresh2_sel"
+permutation "Refresh1" "Refresh2" "I1" "Refresh2"  "Refresh2_sel" "Refresh1" "Refresh1_sel"
+
+#after refresh1, refresh2 can run concurrently
+permutation "LockCompleted" "Refresh1" "Refresh2" "UnlockCompleted"
```

Comment:
@JLockerman  I don't think the `.out` has output for this test yet, re-review once it's added.